cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,58 @@
 /// <reference types='cypress' />
 
-describe('Web Tables page', () => {
-  it('', () => {
+describe("Web Tables page visibility ", () => {
+  beforeEach(() => {
+    cy.visit("https://demoqa.com/webtables");
+  });
+
+  it("Should check the visibility of pagination", () => {
+    cy.get(".-pagination").should("be.visible");
+  });
+
+  it("Should check the visibility of Rows count selection option", () => {
+    cy.get(".-pageSizeOptions").should("be.visible");
+  });
```

Comment:
Test case for row count selection should include assertions to verify the change in displayed rows count.